src/models/mod.rs

```diff
@@ -2,15 +2,13 @@
 
 mod envelope;
 mod helpers;
-mod layout;
-mod link;
+pub mod layout;
+pub mod link;
 mod metadata;
 mod predicate;
 mod statement;
 
 pub use helpers::*;
-pub use layout::*;
```

Comment:
I'll leave final approval to @SantiagoTorres, but is this re-exporting now not happy path anymore? Did something change? I ask because we haven't seen this before, right?

Reply:
Also, does this not break any consumers of the library? Must they update their import paths to require `layout` / `link`?

Reply:
you're right. I'll look into how I can resolve it.

it's a clippy warning about re-importing some of the globs.

Reply:
Fixed, I over complicated it. Since the only things exported by the sub-metadata modules are already explicitly exported by `mod.rs` it works now.